internal/services/monitor/monitor_action_group_resource.go

```diff
@@ -476,6 +483,7 @@ func resourceMonitorActionGroupCreateUpdate(d *pluginsdk.ResourceData, meta inte
 	client := meta.(*clients.Client).Monitor.ActionGroupsClient
 	tenantId := meta.(*clients.Client).Account.TenantId
 	subscriptionId := meta.(*clients.Client).Account.SubscriptionId
+	location := d.Get("location").(string)
```

Comment:
We need to normalise this
```suggestion
	location := location.Normalize(d.Get("location").(string))
```

Reply:
Thanks @stephybun, this has been updated as well. Still learning...